components/elm/docs/tech-guide/shortwave_radiation.md

```diff
@@ -0,0 +1,807 @@
+\end{eqnarray}
+$$
+
+### Upscatter for diffuse radiation
+
+The upscatter for diffuse radiation also accounts for the
+presence of snow on vegetation as
+
+$$
+\begin{equation}
+\omega_\Lambda \beta_\Lambda =
+\begin{cases}
+\omega_\Lambda^{veg} \beta_\Lambda^{veg} \left( 1 -
+f_{wet}\right) + \omega_\Lambda^{sno} \beta_\Lambda^{sno}
+f_{wet}, & \text{if snow is on the canopy}\\[0.5em]
+\omega_\Lambda^{veg} \beta_\Lambda^{veg}, &
+\text{otherwise}
+\end{cases}
+\end{equation}
+$$
+
+where
+
+$$
+\begin{eqnarray}
+\omega_\Lambda^{veg} \beta_\Lambda^{veg} &=& \dfrac{1}{2}
+\left[ \alpha_\Lambda + \tau_\Lambda + (\alpha_\Lambda -
+\tau_\Lambda) \cos^2\bar{\theta}\right] \nonumber
+\\[0.5em]
+&=& \dfrac{1}{2} \left[ \alpha_\Lambda + \tau_\Lambda +
+(\alpha_\Lambda - \tau_\Lambda) \left( \dfrac{1 +
+\chi_\ell}{2} \right)^2\right]
+\end{eqnarray}
+$$
+
+### Upscatter for direct radiation
+
+The upscatter for direct radiation accounts for the
+presence of snow on vegetation as
+
+$$
+\begin{equation}
+\omega_\Lambda \beta_{0,\Lambda} =
+\begin{cases}
+\omega_\Lambda^{veg} \beta_{0,\Lambda}^{veg} \left( 1 -
+f_{wet}\right) + \omega_\Lambda^{sno}
+\beta_{0,\Lambda}^{sno} f_{wet}, & \text{if snow is on the
+canopy}\\[0.5em]
+\omega_\Lambda^{veg} \beta_{0,\Lambda}^{veg}, &
+\text{otherwise}
+\end{cases}
+\end{equation}
+$$
+
+where
+
+$$
+\begin{eqnarray}
+\omega_\Lambda^{veg} \beta_{0,\Lambda}^{veg} &=&
+\left(\dfrac{1 + \bar{\mu}K}{\bar{\mu}K} \right)
+a_s(\mu)_\Lambda \nonumber \\[0.5em]
+&=& \left(\dfrac{1 + \bar{\mu}K}{\bar{\mu}K}\right)
+\dfrac{\omega_\Lambda^{veg}}{2} \dfrac{G(\mu)}{\mu \phi_2 +
+G(\mu)}
+\left[ 1 - \dfrac{\mu \phi_1}{\mu \phi2 + G(\mu)} \ln
+\left( \dfrac{\mu \phi_1 + \mu \phi_2 + G(\mu)}{\mu \phi1}
+\right) \right]
+\end{eqnarray}
+$$
+
+## Analytical Solutions of the Two-Stream Approximation Radiative Transfer Model
+
+The analytical solutions for upward and downward radiation
+at a depth equal to the cumulative leaf and stem area
+index, $\kappa$, were derived by Sellers (1985) for the
+two-stream approximation model given by equations
+\eqref{eqn:dif_up}-\eqref{eqn:dif_dn} under a set of
+prescribed boundary conditions.
+
+### Scattered Beam Radiation
+
+The following boundary conditions are used for the values
+of downward diffuse radiation per unit incident direct
+radiation at the top of the canopy (i.e., $\kappa = 0$)
+and upward diffuse per unit incident direct radiation at
+the bottom of the canopy (i.e., $\kappa = L + S$):
+
+$$
+\begin{eqnarray}
+I^{\downarrow}_{b,\Lambda}(\kappa = 0) &=& 0 \\[0.5em]
+I^{\uparrow}_{b,\Lambda}(\kappa = L + S) &=&
+\alpha_{gb,\Lambda} e^{-K(L+S)} +
+\alpha_{gd}I^{\downarrow}_{b,\Lambda}
+\end{eqnarray}
+$$
+
+with $\alpha_{gb,\Lambda}$  and $\alpha_{gd,\Lambda}$ are
+the ground albedo for direct and diffuse radiation,
+respectively. Under such boundary conditions, the
+analytical solution of upward and downward diffuse
+radiation per unit incident direct radiation is then given
+as
+
```

Comment:
Suggest replacing all occurences of phrases like "is then given as by "is", and of phrase like "are as follows" by "are".